parameter/module.go

```diff
@@ -22,8 +22,4 @@ package parameter
 
 import "go.uber.org/fx"
 
-var Module = fx.Options(
-	fx.Provide(
-		StoreFromObjectStore,
-	),
-)
+var Module = fx.Options()
```

Comment:
I am unable to understand why we would want to initialize this here?  The actual parameter implementation needs to come from a different library anyway.  Would it be okay to remove this and the initialization in the orlop/module.go line 14?